app/javascript/react/screens/App/Overview/components/AggregateCards/ActiveTransformationPlans/ActiveTransformationPlans.js

```diff
@@ -22,7 +23,18 @@ const ActiveTransformationPlans = ({ activePlans, allRequestsWithTasks, reloadCa
       const mostRecentRequest = requestsOfAssociatedPlan.length > 0 && getMostRecentRequest(requestsOfAssociatedPlan);
       return (
         mostRecentRequest &&
-        mostRecentRequest.miq_request_tasks.some(task => task.state === 'finished' && task.status === 'Error')
+        mostRecentRequest.miq_request_tasks.some(task => {
+          if (task.options && task.options.playbooks) {
```

Comment:
I believe `.options` is always gonna be present... no need to do the precheck... if NOT then we oughta revisit https://github.com/ManageIQ/miq_v2v_ui_plugin/pull/442/files#r204044128 as well